src/CSLOLSystemTrayManager.cpp

```diff
@@ -0,0 +1,86 @@
+#include "CSLOLSystemTrayManager.h"
+#include <QQmlContext>
+#include <QDesktopServices>
+#include <QUrl>
+#include <QCoreApplication>
+#include <QSystemTrayIcon>
```

Comment:
This depends on Qt widgets.
We are not building with Qt widgets enabled in cmake, so this will fail.
In fact i'm not sure if our build of Qt even has widgets enabled: https://github.com/LeagueToolkit/cslol-manager/blob/master/.github/workflows/qt.yaml#L16

Reply:
Fair enough. Is this the build of Qt that the buildit workflow uses? I only ask because that was what I was using to test the commits.